test/browser/features/fixtures/auto_notify/script/a.html

```diff
@@ -8,7 +8,7 @@
       var API_KEY = decodeURIComponent(window.location.search.match(/API_KEY=([^&]+)/)[1])
       var bugsnagClient = bugsnag({
         apiKey: API_KEY,
-        endpoints: { notify: ENDPOINT },
+        endpoints: { notify: ENDPOINT, sessions: '/noop' },
```

Comment:
Loads of the end-to-end tests that relied on being allowed to not set a custom session endpoint now need some value to satisfy the config validation, but all of these tests only make assertions about errors.

I've set it to this value here (and many other places), which hopefully which suggests it's not used at all to appease the config validation.